Makefile

```diff
@@ -98,6 +98,24 @@ run-dev-image:
 		-v $(CURDIR)/collectd:/usr/src/collectd:cached \
 		signalfx-agent-dev /bin/bash
 
+.PHONY: run-k8s-tests
+run-k8s-tests:
+	docker rm -f -v minikube registry || true
```

Comment:
Shouldn't this create them instead of deleting them?

Reply:
The minikube and registry containers are auto-created by the fixtures. This just makes sure that any left-over containers are removed before the tests start.